back/engines/commercial/analysis/spec/acceptance/taggings_spec.rb

```diff
@@ -70,28 +70,27 @@
   end
 
   post 'web_api/v1/analyses/:analysis_id/taggings/bulk_create' do
-    with_options scope: :tagging do
-      parameter :tag_id, 'The ID of the tag to associate the input with', required: true
-      parameter :filters, 'The filters to select the inputs to associate the tag with', required: true
-    end
+    parameter :tag_id, 'The ID of the tag to associate the input with', required: true
+    parameter :filters, 'The filters to select the inputs to associate the tag with', required: true
+
     ValidationErrorHelper.new.error_fields(self, Analysis::Tagging)
 
     let(:analysis) { create(:analysis) }
     let(:analysis_id) { analysis.id }
-    let(:tag_id) { create(:tag, analysis: analysis).id }
     let!(:inputs) { create_list(:idea, 3, project: analysis.project) }
 
-    example_request 'Bulk create taggings' do
-      tag_id= create(:tag, analysis: analysis).id
-
-
-      do_request(tagging: { tag_id: tag_id, filters: { search: "" } })
-
+    example 'Bulk create taggings' do
+      tag_id = create(:tag, analysis: analysis).id
+      idea = create(:idea, project: analysis.project, title_multiloc: {
+        'en' => 'My idea',
+        'fr-FR' => 'Mon idée',
+        'nl-NL' => 'Mijn idee'
+      })
```

Comment:
Small suggestion: Also add a 2nd idea that is already assigned to the tag and matches the filters too.
```
idea2 = create(:idea, project: analysis.project)
create(:tagging, input: idea2, tag_id: tag_id, title_multiloc: {'en' => 'My idea is great'})
```